views/index.ejs

```diff
@@ -70,6 +70,12 @@
   <p class="error-notice">If you find any errors in feature detection, please <a href="https://github.com/foolip/mdn-bcd-collector/issues/new"><span class="mdi mdi-alert-circle-outline"></span> file an issue</a> in the GitHub repository.</p>
 </div>
 
+<div id="privacy-notice" class="paper">
+  <h2><span class="mdi mdi-alert"></span> Privacy Notice</h2>
+  <p>This website is designed to test as many browser features as possible. This includes features revealing personal information, such as microphone, camera, geolocation, etc. <strong>No personal information is stored locally or transmitted to any servers.</strong></p>
+  <p>We also use cookies to track your session and link the test results to your session.</p>
```

Comment:
```suggestion
  <p>Cookies are used for session tracking, so that test results can be stored on the server and then optionally exported. Test results are deleted after 15 days.</p>
```